### PR TITLE
Added task "Fix infinite loop" to cover infinite loop concept

### PR DESCRIPTION
**Pull request by sofiiako**

### CODE REVIEW

File: Loops/Fix infinite execution/tests/test_task.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import unittest
+import contextlib
+import io
+
+import timeout_decorator
```
Comment (Geravant) [inline Loops/Fix infinite execution/tests/test_task.py:5]: I've got this while running the checker

Error
Traceback (most recent call last):
  File "C:\Users\Igor.Gerasimov\introduction_to_python\lib\site-packages\timeout_decorator\timeout_decorator.py", line 75, in new_function
    old = signal.signal(signal.SIGALRM, handler)
AttributeError: module 'signal' has no attribute 'SIGALRM'

Something seems off with the lib

Comment (Geravant) [reply Loops/Fix infinite execution/tests/test_task.py:5]: Seems like we have to look for a workaround

**Review by Geravant — APPROVED** [review]

File: Loops/Fix infinite execution/task.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+## Fix infinite execution
+
+In the code editor, there is a `while` loop, defined inside a [function](course://Functions/Definition). Please do not worry about it at the moment,
```
Comment (stephen-hero) [inline Loops/Fix infinite execution/task.md:3]: In the code editor, there is a `while` loop defined inside a [function](course://Functions/Definition). Please do not worry about it at the moment,

**Review by stephen-hero — COMMENTED** [review]
one sentence restructured, one extra comma deleted

**Pull request merged**